src/reducers/setCursorInitialized.ts

```diff
@@ -0,0 +1,17 @@
+import { Path, State } from '../@types'
+
+/**
+ * Set cursor initialized status and the cursor path after initial pull.
+ */
+const setCursorInitialized = (
+  state: State,
+  { cursor, cursorInitialized }: { cursor?: Path | null; cursorInitialized: boolean },
+) => {
+  return {
+    ...state,
+    cursor,
```

Comment:
I'm not sure it's safe to change the cursor directly. We typically dispatch `setCursor` which keeps edit mode, `expanded`, and other cursor-related state in sync. Can we add an `init: boolean` option to `setCursor` instead of creating a separate reducer?

Reply:
Sure, that seems reasonable. I will remove this new reducerand add `init` to `setCursor`